modules/rostests/apitests/user32/RedrawWindow.c

```diff
@@ -425,14 +430,15 @@ UINT TestRedrawWindow2(STRUCT_TestRedrawWindow* ptestRW, STRUCT_TestRedrawWindow
     return countErrors;
 }
 
-void InitRect(RECT *rect, int left, int top, int right, int bottom) {
+void InitRect(RECT *rect, int left, int top, int right, int bottom)
```

Comment:
Not sure why we have this function, because there is already the [SetRect() function] (https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setrect) from user32, that does just that.

Reply:
Thanks for the advice, SetRect works the same as InitRect, so I replaced it.